### ISSUE TITLE
Need a more intelligent way of waiting for NAT Gateways to delete

### ISSUE THREAD

**Issue by SeanAkersGitHub** [SeanAkersGitHub/awsdevops#1]
When tearing down the VPC, the NAT gateways take a long time to get to the 'deleted' status. We can't proceed with anything else until this state is reached. 

issue us in: 
delete_all_assets.sh

**Comment by SeanAkersGitHub** [comment]
done and merged into master
